### PR TITLE
Add Service resource to admin dashboard

### LINKED ISSUE #342: Add Service resource to admin dashboard
# Feature
## Why is this feature being added?

Gives admins the ability to conduct CRUD actions on the `Service` model.  Currently this has to be done through prod's rails console.

## What should your feature do?
Here are the ActiveAdmin docs:  https://activeadmin.info/documentation.html

Here is a sample implementation of a Resource in our codebase:  https://github.com/OperationCode/operationcode_backend/blob/master/app/admin/team_member.rb

- [ ] Adds a Resource for `Service` to ActiveAdmin
- [ ] Updates `ability.rb` with the proper permissions - confirm with @hpjaj

### PR DESCRIPTION
**Pull request by nagano564**
# Description of changes
Gives admins the ability to conduct CRUD actions on the Service model. Currently this has to be done through prod's rails console.

Adds a Resource for Service to ActiveAdmin
Updates ability.rb with the proper permissions

# Issue Resolved

Fixes #342

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -14,12 +14,14 @@ def initialize(user)
...
       can :read, User
       can [:read, :update], AdminUser, id: user.id
+      can :read, Service
 
       return unless user.role&.admin_accessible?
       can :read, :all
       can :manage, CodeSchool
       can :manage, Location
       can :manage, TeamMember
+      can :manage, Serivce
```
Comment (kirbygit) [inline app/models/ability.rb]: Typo error

File: app/models/ability.rb [hunk 2]
```diff
@@ -14,12 +14,14 @@ def initialize(user)
       can :read, TeamMember
       can :read, User
       can [:read, :update], AdminUser, id: user.id
+      can :read, Service
```
Comment (hpjaj) [inline app/models/ability.rb]: Small nit.  Can you pls alphabetize this in both this list, and the one below

File: db/seeds.rb [hunk 3]
```diff
@@ -32,7 +32,13 @@
 SeedTeamMembers.seed_all
 
 # Create Admin (development only)
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
+super_admin = Role.create(title: 'super_admin')
+admin = Role.create(title: 'admin')
+board = Role.create(title: 'board_member')
+
+AdminUser.create!(email: 'super_admin@example.com', password: 'password', password_confirmation: 'password', role_id: 1) if Rails.env.development?
```
Comment (hpjaj) [inline db/seeds.rb]: You'll want to set the `role_id`s dynamically, i.e. `role_id: super_admin.id`

File: yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,32 @@
+Arguments:
```
Comment (hpjaj) [inline yarn-error.log]: You will need to remove this file from this PR.  This cannot be committed.

Comment (hpjaj) [inline yarn-error.log]: You will need to remove this file from this PR. This cannot be committed.

**Pull request merged**